charts/authelia/values.yaml

```diff
@@ -87,7 +88,8 @@ service:
   #   myLabel: myValue
 
   port: 80
-
+  nodePort: 9091
```

Comment:
As far I know, the classic allowed node port is between 30000 and 32767. Is it good to keep 9091?

Reply:
Ah yea good point. Just pick something int hat range. I usually expand the node port range.